src/game/actions.js

```diff
@@ -100,3 +100,4 @@ export const CHANGE_ATTACK = "CHANGE_ATTACK";
 export const CHANGE_HEALTH = "CHANGE_HEALTH";
 export const DAMAGE = "DAMAGE";
 export const SUMMON_CREATURE = "SUMMON_CREATURE";
+export const BUFF_HAND = "BUFF_HAND";
```

Comment:
Hmmmm probably we just need a "buff" action, and this particular battlecry targets it at the hand, yeah?